build.sbt

```
@@ -947,7 +947,9 @@ lazy val std = crossProject(JSPlatform, JVMPlatform, NativePlatform)
       ProblemFilters.exclude[MissingClassProblem]("cats.effect.std.Queue$UnsafeUnbounded$Cell"),
       // introduced by #3480
       // adds method to sealed Hotswap
-      ProblemFilters.exclude[ReversedMissingMethodProblem]("cats.effect.std.Hotswap.get")
+      ProblemFilters.exclude[ReversedMissingMethodProblem]("cats.effect.std.Hotswap.get"),
+      ProblemFilters.exclude[DirectMissingMethodProblem](
+        "cats.effect.std.Supervisor.apply$default$2")
```

This filter is highly unfortunate 🤔 if we ever add default parameters to the public API in the future then it will mask bincompat issues there.

Is it possible that this filter is needed only for Scala 3?

---

Yep, it makes sense only for Scala3, I will fix it

---

Also, sorry for stealing your fruit 😄